wagtail/images/tests/test_admin_views.py

```diff
@@ -314,7 +314,7 @@ def test_num_queries(self):
         self.get()
 
         # Initial number of queries.
-        with self.assertNumQueries(15):
+        with self.assertNumQueries(12):
```

Comment:
This performs better than #9594 in this test as well, with one fewer query for each assertion.